minitests/roi_harness/runme.tcl

```diff
@@ -433,15 +446,19 @@ if {$fixed_xdc eq ""} {
     puts "Routing ROI inputs"
     # Arbitrary offset as observed
     set y_left $Y_DIN_BASE
-    set y_right [expr {$Y_DIN_BASE + 24}]
+    set y_right [expr {$Y_DIN_BASE + $Y_OFFSET}]
     for {set i 0} {$i < $DIN_N} {incr i} {
         # needed to force routes away to avoid looping into ROI
         if {[net_bank_left "din[$i]"]} {
             set node "INT_L_X0Y${y_left}/EE2BEG2"
             route_via2 "din_IBUF[$i]" "$node"
             set y_left [expr {$y_left + $PITCH}]
         } else {
-            set node "INT_R_X25Y${y_right}/WW2BEG1"
+            if {$part eq "xc7z010clg400-1"} {
```

Comment:
Rather than a part driven if, just make a new env var?

Reply:
Done. Actually, I have tried to make it the more general as possible by adding more than an env var. I double checked it and it works fine